tklock.c

```diff
@@ -430,9 +347,6 @@ static gboolean ts_event_control(gboolean enable)
 	execute_datapipe(&touchscreen_suspend_pipe, GINT_TO_POINTER(!enable),
 			USE_INDATA, CACHE_INDATA);
 
-	if (generic_x11_event_control(enable))
-		return TRUE;
-
```

Comment:
Why is that chunk removed in a commit named "Split x11 calls into seperate file"? Commit name implies there are no functional changes. Please move that change to a separate commit, along with a relevant commit message.

Reply:
Its not called that its called that the commit is called:  
Disable xinput devices when display is blanked 
Split x11 calls into seperate file

disabeling the touchscreen xinput device is redundant if you also disable all xinput devices.